sys/net/network_layer/sixlowpan/ip.c

```diff
@@ -33,6 +33,7 @@
 #include "lowpan.h"
```

Comment:
Basically yes, but then no routing protocol will work in mainline RIOT until we merge this PR or adjust the other protocols.